src/Controller/MemberController.php

```diff
@@ -315,9 +330,15 @@ public function connect()
                 $this->showConnectionPage("L'email et le mot de passe doivent être renseignés.");
             } else {
                 $member = $this->memberManager->getFromEmail($_POST['email']);
+
                 if ($member !== null) {
-                    if (password_verify($_POST['password'], $member->getPassword())) {
+                    // Brute force protection
+                    if (!WebsiteCop::canConnectAgain()) {
+                        $this->showConnectionPage("Vous vous êtes trompé trop souvent. Attendez un moment pour réfléchir.");
```

Comment:
It could be great to show the user for how long he could not retry

Reply:
Yes indeed... Done!